corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -34,7 +34,7 @@ def process_change(self, pillow_instance, change):
             # the same effect, an app having has_submissions set to True.
             mark_has_submission(domain, build_id)
 
-        user_id = doc.get('user_id')
+        user_id = doc.get('form', {}).get('meta', {}).get('userID')
```

Comment:
i think this'll work for couch, but not sql. sql stores the user id in a property whereas couch stores the entire form in the doc so it gets it from the metadata